sonic-net/sonic-gnmi
# <WIP>Bookworm migration for all dependency packages in azurepipeline for sonic-gnmi

#### Why I did it
As part of the migration of sonic-gnmi repo to bookworm, migrating the azure pipeline files to refer to the bookworm build for dependency installation.

#### How I did it
Changed all references from bullseye to bookworm in azure-pipeline for dependency packages installation.

#### How to verify it
The tests in azurepipeline were run on the bookworm build locally and has the same behavior as bullseye build.

**Note:** These changes should be merged after the below PR to migrate the sonic-gnmi and sonic-mgmt-framework build to bookworm is merged.
https://github.com/sonic-net/sonic-buildimage/pull/18548

#### Which release branch to backport (provide reason below if selected)

#### Description for the changelog

#### Link to config_db schema for YANG module changes

#### A picture of a cute animal (not mandatory but encouraged)
